src/helpers.c

```diff
@@ -995,3 +995,13 @@ bool is_compatible_format(int format_num)
 	free(format_manifest);
 	return ret;
 }
+
+/* Check if the version argument is the same version as the current OS */
+bool is_current_version(int version)
+{
+	if (version < 0) {
+		return false;
+	}
+
+	return (version == get_current_version(path_prefix));
```

Comment:
Can `get_current_version()` return an error at this point in the code? If so, I would handle its error before deciding how to return from this function.

Reply:
well at this point get_current_version can return a -1, but version cannot be -1. So if get_current_version encounters an error (version == get_current_version(...)) will evaluate to false.

Reply:
Okay, sounds good. I was lacking context of the surrounding code, so I think you can keep it as is.